github_pr.py

```diff
@@ -149,6 +149,47 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers(approved_users, comment_users):
+    comment_approved_users = [comment_user for comment_user in comment_users if comment_user in approved_users]
+    if not comment_approved_users:
+        raise AssertionError("No approved mergers were found in comments - Approved mergers: %s" % approved_users)
+    else:
+        return comment_approved_users
+
+
+def _check_approved_mergers_file(approved_users_file, comment_users):
```

Comment:
you could call `_check_approved_mergers()` instead of lines 163-167. they seem to be an exact duplicate